src/mmrelay/migrate.py

```
@@ -1578,12 +1627,44 @@ def migrate_database(
+                logger.info("Removing destination-only database sidecar: %s", dest)
+                if dest.is_dir():
+                    _retry_on_file_in_use(
+                        lambda d=dest: shutil.rmtree(str(d), ignore_errors=False),
+                        f"rmtree {dest}",
+                    )
+                else:
+                    _retry_on_file_in_use(
+                        lambda d=dest: d.unlink(),
+                        f"unlink {dest}",
+                    )
+        except Exception as exc:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)